src/start_fact.py

```diff
@@ -64,7 +68,7 @@ def _terminate_process(process):
             process.kill()
 
 
-def shutdown(signum, frame):
+def shutdown(*_):
```

Comment:
Consider adding `**__` to avoid running into runtime issues

Reply:
this should not be an issue since it was like this in `start_fact_frontend.py` for several months without problems